doc/source/arf.rst

```diff
@@ -111,6 +111,18 @@ Types, macros and constants
     test code. If in doubt, simply try with some convenient high precision
     instead of using this special value, and check that the result is exact.
 
+.. macro:: nn_mul_2x1(r2, r1, r0, a1, a0, b0)
+
+    Sets `(r_2, r_1, r_0)` to `(a_1, a_0)` multiplied by `b_0`.
```

Comment:
Might want to document that these are limbs. Also, I think these macros don't allow aliasing, which might be worth documenting.